src/pages/addItem.test.js

```diff
@@ -213,6 +220,32 @@ describe('<AddItem/>', () => {
                 screen.getByTestId('saving-exchange-loader')
             ).toBeInTheDocument()
         )
+
+        const expectedPayload = {
+            name: 'an exchange name',
+            source: {
+                requests: [
+                    {
+                        dx: ['anIdDe'],
+                        filters: [],
+                        inputIdScheme: 'UID',
+                        name: 'a request name',
+                        ou: ['anIdOu'],
+                        outputIdScheme: 'UID',
+                        pe: ['anIdPe'],
+                    },
+                ],
+            },
+            target: {
+                request: {
+                    idScheme: 'UID',
+                },
+                type: 'INTERNAL',
+            },
+        }
+
+        expect(createExchangeMock).toHaveBeenCalled()
+        expect(createExchangeMock).toHaveBeenCalledWith(expectedPayload)
```

Comment:
I've updated this test to check the payload being sent on save (which seems reasonable in general), and also has the benefit of confirming that the "NONE" ID Schemes that are selected by default get filtered out.